<?xml version="1.0"?>
<launch>

    <node pkg="uwb_imu" type="imu_uwb_fusion_ukf_node" name="ukf_node" output="screen">
        <!-- <param name="sigma_an" value="3.5848651612538265e+04"/> 
        <param name="sigma_wn" value="5.0319853834530663e-00"/>
        <param name="sigma_aw" value="1.4189758078282432e-03"/>
        <param name="sigma_ww" value="1.3487170893986536e-05"/> -->
        <param name="sigma_an" value="0.01"/> 
        <param name="sigma_wn" value="0.05"/>
        <param name="sigma_aw" value="0.0000001"/>
        <param name="sigma_ww" value="0.0001"/>
    </node>
    
    <node pkg="uwb_imu" type="imu_uwb_fusion_node" name="esekf_node" output="screen">
        <param name="sigma_an" value="3.5848651612538265e+04"/> 
        <param name="sigma_wn" value="5.0319853834530663e-00"/>
        <param name="sigma_aw" value="1.4189758078282432e-03"/>
        <param name="sigma_ww" value="1.3487170893986536e-05"/>
        <!-- <param name="sigma_an" value="0.01"/> 
        <param name="sigma_wn" value="0.05"/>
        <param name="sigma_aw" value="0.0000001"/>
        <param name="sigma_ww" value="0.0001"/> -->
    </node>

    <node pkg="uwb_imu" type="moving_average_filter_node" name="moving_average_filter_node" output="screen">
    </node> 

    <!-- <node pkg="uwb_imu" type="uwb_filtered" name="uwb_filtered" output="screen">
    </node>   -->
    <node name="traj_plot_node" pkg="uwb_imu" type="traj_plot.py" output="screen"/>
    <node name="uwb_node" pkg="uwb_imu" type="uwb.py" output="screen"/>
    <!-- <node name="um7_node" pkg="um7" type="traj_plot.py" output="screen"/> -->
    <!-- <node pkg="uwb_imu" type="uwb_plot_node" name="uwb_plot_node" output="screen">
    </node> -->
	
</launch>

